ui/main.qml

```diff
@@ -506,7 +506,11 @@ StatusWindow {
                     Global.addCentralizedMetricIfEnabled("usage_data_shared", {placement: Constants.metricsEnablePlacement.onboarding})
                 }
             }
-            onCurrentPageNameChanged: Global.addCentralizedMetricIfEnabled("navigation", {viewId: currentPageName})
+            onCurrentPageNameChanged: {
+                if (currentPageName !== "") {
```

Comment:
Hmm yeah that's actually true, I wonder if (when transitioning between pages), the `currentPageName` might be empty :thinking:

Reply:
But, since we now use nested stack views, I wonder if the current approach of getting the page name is sufficient @micieslak :
```qml
readonly property string currentPageName: stack.topLevelItem ? Utils.objectTypeName(stack.topLevelItem) : ""
```

Reply:
It seems to be sufficient as it's based on `topLevelItem`, not just stack's `currentItem`. However the page can be wrapped in a `Loader` and it should be taken into account, e.g.:
```qml
    readonly property string currentPageName: {
        if (!stack.topLevelItem)
            return ""
        
        const item = stack.topLevelItem instanceof Loader ? stack.topLevelItem.item
                                                          : stack.topLevelItem
        
        return Utils.objectTypeName(item)
    }
```

SB page for onboarding layout can be used as a reference bc the stack content discovery works well there.

Reply:
Thanks @micieslak @caybro, I updated `currentPageName` calculation